gammapy/estimators/excess_map.py

```diff
@@ -128,6 +128,10 @@ def __init__(
         self.selection_optional = selection_optional
         self.energy_edges = energy_edges
         self.correlate_off = correlate_off
+
+        if spectral_model is None:
+            spectral_model = PowerLawSpectralModel()
```

Comment:
Maybe explicitly pass `index=2` in case the default is changed?

Reply:
OK